product_docs/docs/pgd/5/rel_notes/pgd_5.3.0_rel_notes.mdx

```diff
@@ -0,0 +1,77 @@
+| BDR       | 5.3.0   | Fixed unsafe CAMO decisions in remote_write mode. ||
+| BDR       | 5.3.0   | Taskmgr process now respects SIGINT. ||
+| BDR       | 5.3.0   | Speed up manager process startup by limiting the amount of WAL read for loading commit decisions. ||
+| BDR       | 5.3.0   | Physical joins now clean up stale records in bdr.taskmgr_local_work_queue. ||
+| BDR       | 5.3.0   | Fixed a bug in copying bdr.autopartition_rules during logical join. ||
+| BDR       | 5.3.0   | Override bdr.ddl_replication=off in taskmgr worker. ||
+| BDR       | 5.3.0   | Avoid a potential deadlock between `bdr.autopartition_wait_for_partitions()` and taskmgr||
+| BDR       | 5.3.0   | Fixed writer missing updates in streaming mode with TDE enabled. ||
+| BDR       | 5.3.0   | Block new EPAS automatic partitioning on PGD cluster. ||
+| BDR       | 5.3.0   | Allow existing automatically partitioned tables when cluster is created or upgraded. ||
+| BDR       | 5.3.0   | Block PGD Autopartition on EPAS INTERVAL partitioned table. ||
+| BDR       | 5.3.0   | Ensure that replication does not break when bdr.autopartition() is invoked on a mixed version cluster, running with 3.7.23 and 4.3.3/5.3 ||
+| BDR       | 5.3.0   | Fixed default selective replication behavior for data groups. The data groups are supposed to publish only replication sets of the group or any parent groups by default, which mirrors to what they subscribe. ||
+| BDR       | 5.3.0   | Fixed memory leak in LCR file TDE encryption. ||
+| BDR       | 5.3.0   | Fixed memory leak in streaming transaction processing. ||
+| BDR       | 5.3.0   | Allow force parting of nodes that are already being parted normally. ||
+| BDR       | 5.3.0   | PART_CATCHUP is now more resilient to replication slot and node_catchup_info conflict. ||
+| BDR       | 5.3.0   | Fixed row filter failure for partitions created after a table column was dropped | RT95149 |
+| BDR       | 5.3.0   | Avoid aborting a group commit transaction on receiving the first abort response, the other nodes are given a chance to respond and transaction can succeed if the required responses are received ||
+| BDR       | 5.3.0   | Ensure that replication receiver worker reloads configuration correctly when `pg_reload_conf()` is called. ||
+| BDR       | 5.3.0   | Prevent duplicate Raft request ID generation which could break replication. ||
+| BDR       | 5.3.0   | Fixed several rare memory access issues that could potentially cause crashes of workers. ||
+| BDR       | 5.3.0   | Fixed issue where explicit 2PC transactions would abort earlier when encountering conflicts. | RT92897 |
+| CLI       | 5.3.0   | Fixed `verify-settings` command if the `shared_preload_libraries` GUC contains file path ||
+| CLI       | 5.3.0   | Show replslots status as `Critical` in `check-health` command when PGD cluster is reduced just a witness node | |
```

Comment:
Should the phrase be `is reduced to a witness node`? Please note the missing `to`.

Reply:
```suggestion
| CLI       | 5.3.0   | Show replslots status as `Critical` in `check-health` command when PGD cluster is reduced to just a witness node | |
```